src/charm.py

```diff
@@ -68,7 +68,31 @@ class PgBouncerK8sCharm(CharmBase):
     def __init__(self, *args):
         super().__init__(*args)
 
-        self.secrets = {APP_SCOPE: {}, UNIT_SCOPE: {}}
+        self.peer_relation_app = DataPeer(
+            self,
+            relation_name=PEER_RELATION_NAME,
+            additional_secret_fields=[
+                "monitoring-password",
+                "operator-password",
+                "replication-password",
+                "rewind-password",
```

Comment:
Does this operator have all these passwords?

Reply:
Not really. Will check tomorrow what needs to be kept and what doesn't.

Reply:
Pgbouncer needs arbitrary secret names, which are currently not supported by the lib. The PR will have to wait until there is a way to do this or when we know we need to change this.

Reply:
List should be up to date now.